<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GidPyQtSorterMain</class>
 <widget class="QMainWindow" name="GidPyQtSorterMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>1123</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>[Gid] PyQt-Sorter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/sidecar_1</normaloff>:/icons/sidecar_1</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTabWidget" name="main_output_tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="actionList"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionOpen_Ui_File"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionMake_Boiler_File"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOpen_Settings"/>
    <addaction name="actionOpen_Color_Dialog"/>
   </widget>
   <widget class="QMenu" name="menuExtensions">
    <property name="title">
     <string>Extensions</string>
    </property>
    <addaction name="actionAdd_Extension"/>
    <addaction name="actionManage_Extensions"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuExport"/>
   <addaction name="menuSettings"/>
   <addaction name="menuExtensions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Ui_File">
   <property name="text">
    <string>Open Ui File</string>
   </property>
  </action>
  <action name="actionMake_Boiler_File">
   <property name="text">
    <string>Make Boiler File</string>
   </property>
  </action>
  <action name="actionOpen_Settings">
   <property name="icon">
    <iconset resource="../pyqtsocius/ui_elements/pyqt_sorter_ressources.qrc">
     <normaloff>:/icons/ui_ressources/icons/tools.png</normaloff>:/icons/ui_ressources/icons/tools.png</iconset>
   </property>
   <property name="text">
    <string>Open Settings</string>
   </property>
  </action>
  <action name="actionOpen_Color_Dialog">
   <property name="icon">
    <iconset resource="../pyqtsocius/ui_elements/pyqt_sorter_ressources.qrc">
     <normaloff>:/icons/ui_ressources/icons/color-palette.png</normaloff>:/icons/ui_ressources/icons/color-palette.png</iconset>
   </property>
   <property name="text">
    <string>Open Color Dialog</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>List</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAdd_Extension">
   <property name="text">
    <string>Add Extension</string>
   </property>
  </action>
  <action name="actionManage_Extensions">
   <property name="text">
    <string>Manage Extensions</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../pyqtsocius/ui_elements/pyqt_sorter_ressources.qrc"/>
 </resources>
 <connections/>
</ui>
